RetailCoder.VBE/Navigation/CodeExplorer/CodeExplorerViewModel.cs

```diff
@@ -558,7 +558,7 @@ private void ExecuteRemoveComand(object param)
 
         public Visibility ExportAllVisibility => CanExecuteExportAllCommand ? Visibility.Visible : Visibility.Collapsed;
 
-        public bool IsSourceControlEnabled => _generalSettings.IsSourceControlEnabled;
+        public bool EnableExperimentalFeatures => _generalSettings.EnableExperimentalFeatures;
```

Comment:
hmm, I think we should list all "experimental features," and allow enabling/disabling them selectively. this way you can have experimental source control without the experimental code panes... or vice-versa.

Reply:
That'd be nice, but pretty impossible without adding an attribute for each feature, and then it gets where we might as well leave it the way it currently is.

Reply:
I'm not following. We could put an `[Experimental("Feature_ResourceKey")]` attribute on top of a class, and the settings viewmodel would reflect and locate all the classes with that attribute, get the corresponding resource string, and stick that into the settings tab for experimental features, no?